runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -3682,6 +3682,12 @@ J9::Z::CodeGenerator::suppressInliningOfRecognizedMethod(TR::RecognizedMethod me
       return true;
       }
 
+   if (method == TR::java_util_zip_CRC32C_updateBytes ||
```

Comment:
If we do not have vector support on the platform, there is no need to suppress inlining of this method. In that case, it is actually beneficial to inline.